float32[] laserScan
bool terminal
float32 reward

float32 current_x
float32 current_y
float32 target_x
float32 target_y

float32 desired_x
float32 desired_y
